### PR TITLE
Feature/landing page part 1

### PR DESCRIPTION
**Pull request by yryskeldi0101**

### CODE REVIEW

**Comment by Begimai7** [comment]
checked by me

**Comment by FrontenderO07** [comment]
No problem!

**Comment by Naz-Erke18** [comment]
checked!

File: src/components/landing-page/LandingPage.jsx
Comment (songhee24) [inline src/components/landing-page/LandingPage.jsx:1]: landing page бул containers папкада болуш керек

**Review by songhee24 — CHANGES_REQUESTED** [review]

**Review by songhee24 — APPROVED** [review]

**Pull request merged**